src/main/ErrorMessages.h

```diff
@@ -11,7 +11,12 @@ constexpr auto const REPORT_INTERNAL_BUG =
     "Please report this bug along with this log file if this was not expected";
 constexpr auto const POSSIBLY_CORRUPTED_HISTORY =
     "One or more of history archives may be corrupted. Update HISTORY "
-    "configuration entry to only contain valid ones";
+    "configuration entry to only contain valid ones. If you have a valid "
```

Comment:
Sorry, I wasn't clear: we can't modify POSSIBLY_CORRUPTED_HISTORY directly, because it's used in other scenarios, where the rest of this message isn't relevant/confusing. Please keep this message as-is, then in `CheckSingleLedgerWork` add a log:
```
If this occurs often, consider notifying the archive owner. As long as your configuration has any valid history archives, this error does NOT mean your node is unhealthy.
```